src/renderer/global.scss

```diff
@@ -145,28 +145,28 @@ body {
         background: rgb(203 213 224 / 66%) !important;
 
         &:not(:disabled):hover {
-            background: var(--gray-400) !important;
-            border: 0 solid var(--gray-500) !important;
+            background: var(--theme-400) !important;
```

Comment:
Why did you rename these variables to `--theme-xxx`?

Reply:
Because them being named "gray" is misleading

Reply:
Why is it misleading? Aren't they shades of gray tinted towards a certain color?

Reply:
Not necessarily anymore. I'd much rather reserve the gray variables for things actually trying to be gray than things that could be any color.

Reply:
I guess.